### PR TITLE
Use existing library for filtering files using glob patterns

### PR DESCRIPTION
**Pull request by lpellegr**
Linked with https://github.com/ow2-proactive/programming/pull/1

### CODE REVIEW

**Comment by lpellegr** [comment]
@fviale, could you please review and try to run the tests on Windows?

On Windows, I have succeeded to run tests individually from Intellij if I pass the property "-Djava.library.path". However, if I run all the tests from gradle, some are failing in scheduler-server. The error message I get is the following:

```
java.lang.NoClassDefFoundError: Could not initialize class org.ow2.proactive.rm.util.process.ProcessTreeKiller$Windows
	at org.ow2.proactive.rm.util.process.ProcessTreeKiller.get(ProcessTreeKiller.java:140)
	at org.ow2.proactive.scheduler.task.forked.TaskProcessTreeKiller.kill(TaskProcessTreeKiller.java:61)
	at org.ow2.proactive.scheduler.task.nativ.NativeExecutable.execute(NativeExecutable.java:363)
	at org.ow2.proactive.scheduler.task.TaskLauncher$ExecutableGuard$9.call(TaskLauncher.java:1796)
	at org.ow2.proactive.scheduler.task.TaskLauncher$ExecutableGuard$9.call(TaskLauncher.java:1791)
	at org.ow2.proactive.scheduler.task.utils.ActiveObjectExecutor.call(ActiveObjectExecutor.java:102)
	at sun.reflect.GeneratedMethodAccessor36.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.objectweb.proactive.core.mop.MethodCall.execute(MethodCall.java:353)
	at org.objectweb.proactive.core.body.request.RequestImpl.serveInternal(RequestImpl.java:214)
	at org.objectweb.proactive.core.body.request.RequestImpl.serve(RequestImpl.java:160)
	at org.objectweb.proactive.core.body.BodyImpl$ActiveLocalBodyStrategy.serveInternal(BodyImpl.java:552)
	at org.objectweb.proactive.core.body.BodyImpl$ActiveLocalBodyStrategy.serve(BodyImpl.java:485)
	at org.objectweb.proactive.core.body.AbstractBody.serve(AbstractBody.java:426)
	at org.objectweb.proactive.Service.blockingServeOldest(Service.java:206)
	at org.objectweb.proactive.Service.blockingServeOldest(Service.java:181)
	at org.objectweb.proactive.Service.fifoServing(Service.java:146)
	at org.objectweb.proactive.core.body.ActiveBody$FIFORunActive.runActivity(ActiveBody.java:336)
	at org.objectweb.proactive.core.body.ActiveBody.run(ActiveBody.java:175)
	at java.lang.Thread.run(Thread.java:745)
```

 I cannot figure out what is the cause.

File: build.gradle [hunk 1]
```diff
@@ -21,7 +21,8 @@ task extractNativeBundle(type: Copy) {
 
 buildscript {
     repositories {
-        if (project.hasProperty('local')) mavenLocal()
+//        if (project.hasProperty('local'))
```
Comment (lpellegr) [inline build.gradle]: Has to be uncommented once the patch has been reviewed

**Comment by lpellegr** [comment]
We have discussed about that with Fabien and it seems better to keep the name as it is to prevent any issue with executing code that relies on this class.

**Comment by youribonnaffe** [comment]
That is going to be an interesting one to merge with Speed branch :D

**Pull request merged**